daemons/attrd/attrd_corosync.c

```diff
@@ -543,18 +543,35 @@ attrd_peer_sync(crm_node_t *peer, xmlNode *xml)
     free_xml(sync);
 }
 
+static void
+copy_attrs(xmlNode *src, xmlNode *dest)
+{
+    /* Copy attributes from the wrapper parent node into the child node.
+     * We can't just use copy_in_properties because we want to skip any
+     * attributes that are already set on the child.  For instance, if
+     * we were told to use a specific node, there will already be a node
+     * attribute on the child.  Copying the parent's node attribute over
+     * could result in the wrong value.
+     */
+    for (xmlAttrPtr a = pcmk__xe_first_attr(src); a != NULL; a = a->next) {
+        const char *p_name = (const char *) a->name;
+        const char *p_value = ((a == NULL) || (a->children == NULL)) ? NULL :
+                              (const char *) a->children->content;
+
```

Comment:
An alternate approach here could be to add an overwrite argument to copy_in_properties.  That function also obeys ACLs while this block does not.  I don't think that's important in this context, but still.